features/index.ts

```diff
@@ -38,6 +38,7 @@ export * from "./admin.extensions.v1/public-api";
 export * from "./admin.groups.v1/public-api";
 export * from "./admin.identity-providers.v1/public-api";
 export * from "./admin.identity-verification-providers.v1/public-api";
+export * from "./admin.logs.v1/public-api";
```

Comment:
Shall we not add exports for new modules in the index file

Reply:
This will be required for the `rollup` build.